responses.go

```diff
@@ -253,7 +253,7 @@ type RespSync struct {
 
 	DeviceLists    DeviceLists       `json:"device_lists"`
 	DeviceOTKCount OTKCount          `json:"device_one_time_keys_count"`
-	FallbackKeys   []id.KeyAlgorithm `json:"device_unused_fallback_key_types"`
+	FallbackKeys   []id.KeyAlgorithm `json:"device_unused_fallback_key_types,omitempty"`
```

Comment:
That looks a bit sus

Reply:
It’s optional in the spec I believe, for non e2ee clients I’d guess?

Reply:
It's required in the spec, although it won't exist in servers that don't support the spec version that added it. Is the omitempty needed for something?

Reply:
Oh my bad - doesn't show required under the sync spec but does under OTK, will remove this as was only making the change for spec compliance.